### ISSUE TITLE
FIO Doesn't Preallocate Files In Parallel

### ISSUE THREAD

**Issue by iwoloschin** [axboe/fio#595]
Using fio-3.6, it appears that pre-allocation is not done in parallel, instead it's done one file at a time, even if each fio file is on a separate disk.  Can this be modified to run in parallel?

As a workaround I suspect I can just preallocate the files myself (does fio care if it's a sparse file?), but it'd be nice if this was just builtin to fio.

**Comment by axboe** [comment]
That's on purpose, to avoid crappy file systems from fragmenting files across jobs. But it's also an option, so just set ```create_serialize=0``` and it won't do that. You could always create the files yourself, that's totally fine. If fio sees the files there and they are big enough, it will just use them as-is.

**Comment by iwoloschin** [comment]
Ah, I didn't realize that's already an option, but that'll probably do what I want, thanks!
